content/NIAC.md

```diff
@@ -117,13 +117,27 @@ executive officers:
 All of these officers will be elected for a two year term. The incumbent
 chair and executive secretary are limited to two consecutive terms.
 
-Voting: Only members of the NeXus International Advisory Committee will be able to vote on specific proposals that come under the remit of the committee. However, all debates will be conducted publicly and non-committee members are both allowed and encouraged to participate.  
+Voting: Only members of the NeXus International Advisory Committee will 
+be able to vote on specific proposals that come under the remit of the 
+committee. However, all debates will be conducted publicly and 
+non-committee members are both allowed and encouraged to participate.  
+
 Topics voted on will require a simple majority unless specifically
 challenged by a committee member. In the event of a challenge, the issue
 will be presented to the NeXus mailing list inviting comments for a
 period of one month. At the end of this period, the challenge can be
 overruled by a two thirds majority.
 
+Email voting as used up to 2018 only for membership renewals.
```

Comment:
This appears to describe history and is not a statement for the constitution.  This does not describe the email voting process.  We should add a statement that describes the email voting process.

Reply:
Specifically, we should describe the process for membership renewals.

Reply:
We'll publish as written and work on a revision for the final NIAC2020 close out session.

Reply:
I agree that this statement looks out of place. The constitution should only give the current rules and not comment on previous rules.

Reply:
Agree this should be more clear. Added the words as approved by NIAC2018.  We should propose changes to the close out session of NIAC2020.